modules/jaeger-rn-known-issues.adoc

```diff
@@ -22,4 +22,6 @@ These limitations exist in Jaeger:
 
 These are the known issues in Jaeger:
 
+* link:https://issues.redhat.com/browse/TRACING-1166[TRACING-1166] It is not  possible to use the Jaeger streaming strategy within a disconnected environment. When a Kafka cluster is being provisioned, it results in a error: `Failed to pull image registry.redhat.io/amq7/amq-streams-kafka-24-rhel7@sha256:f9ceca004f1b7dccb3b82d9a8027961f9fe4104e0ed69752c0bdd8078b4a1076`.
```

Comment:
"It is not currently possible" - there was 2 spaces, but thought better to add in "currently", as it will work once the AMQ streaming operator is fixed.

Reply:
done, thanks!